### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Rémy Bleuer

### DISCUSSION THREAD

**Discussion by RemyBlr** [heig-vd-dai-course/heig-vd-dai-course#70] [Show and tell]
[My repository](https://github.com/RemyBlr/dai-bleuer-practical-work-1)
Latest commit hash : d06ec5e

**Comment by ludelafo** [comment]
Thank you! Could you tell us a bit more about the idea you have regarding the practical work?

**Reply by RemyBlr** [reply]
Yes, of course. For the first practical work, I'm planning on doing something simple, just to get the hang of Java and all this new environment. I'm planning to make something that resembles what was shown during the presentation. I will be able to change the input file to all upper/lower case and adding a specific encoding. I'm going to add more commands, for example, reversing a string or alternating letters between upper/lower case. Depending on how it goes,.I will implement even more commands, but I really don't know for for now.
